configuration/items.md

```diff
@@ -202,17 +202,34 @@ Two naming schemes are established in the community for Group names:
 {: #label}
 ### Label
 
-Label text is used to describe an Item in a human-readable way.
-Graphical UIs will display the label text when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
+The label is used to describe an Item in a human-readable way.
+Graphical UIs will display the label when the Item is included, e.g. in [Basic UI]({{base}}/addons/uis/basic/readme.html) in a [Sitemap]({{base}}/configuration/sitemaps.html) definition.
 Some I/O services (e.g. the Amazon Alexa skill) also use the label to match an external voice command to an Item.
 
-In textual configurations the label, in quotation marks, appears next to the optional state presentation field in square brackets (see below).
-The label for the Item in the following example is "Temperature":
+In textual configurations the label in quotation marks, contains the label text and the optional state presentation field in square brackets (see below).
+They will be showed side by side, the text left aligned and the state representation right-aligned.
+The label text for the Item in the following example is "Temperature" and the optional state representation is set to be displayed e.g as "23.9 °C":
 
 ```java
 Number Livingroom_Temperature "Temperature [%.1f °C]"
 ```
 
+The label defined for an Item can be overwritten by the label definition on a sitemap.
+If the label definition on the Item contained label text and the state representation these have to be considered separatly.
```

Comment:
This was really hard to read for me.
Did i get it correct?

1. The item definition is made with some text and a string formatter for the item state
2. We then have to take care of state and "text-label" differently in the sitemap

If yes i would suggest something like:

>If the label defined in an Item contains state and text, these representations have to be considered separately in the sitemap.

Reply:
Please capitalize "Sitemap"

Reply:
This is an important piece of information. I wonder if we did not yet describe this somewhere.

I also feel the explanation is a bit complicated. You can define label text and the state representation in Items and in Sitemaps. Redefiniton in Sitemaps overwrites item definition.

I like to work with absolute names in Item labels and shorter specialized versions in sitemaps.

Reply:
Changed my comment.
A whole pr with capitalization in smarthome open and still doing the same typos...

Reply:
Just for info: https://www.openhab.org/docs/configuration/sitemaps.html#element-types

> When an Item is defined, you have the opportunity to assign a label and/or an icon at that point. If no label or icon are specified in the Sitemap, then the label and/or icon you assigned to the Item will be displayed. Setting a value for label or icon of a Sitemap element will override the values defined for the linked Item.